creator-node/src/fileManager.js

```diff
@@ -295,17 +295,35 @@ async function saveFileForMultihashToFS (serviceRegistry, logger, multihash, exp
     }
 
     // verify that the contents of the file match the file's cid
+    let fileData = null
+    let fileSize = null
     try {
-      decisionTree.push({ stage: 'About to verify the file contents for the CID', vals: multihash, time: Date.now() })
+      fileData = (await fs.readFile(expectedStoragePath)).toString().slice(0, 99)
```

Comment:
these are just to add in the decision tree for extra info

Reply:
1. makes sense, but can we clarify in code that this is just for logging with comment + change varname to indicate it is a `fileDataSlice` otherwise the `slice(0,99)` is very scary haha
2. should clarify that the `fileData` is just for logging, but the `fileSize` is not
3. are you concerned about the added duration of every saveFileForMultihash with these 2 disk calls just for logging? since its just a fixed size slice shoudnt add much, but want to flag
4. also maybe lets parallelize?

Reply:
i'm gonna delete fileData, youre right that its not super useful and probably more trouble than it's worth

Reply:
cool